source/minikube.rst

```diff
@@ -0,0 +1,80 @@
+Install Percona Server for MongoDB on minikube
+==============================================
+
+Installing PSMDB Operator on `minikube <https://github.com/kubernetes/minikube>`_
+is the easiest way to try it locally without a cloud provider. Minikube runs
+Kubernetes on GNU/Linux, Windows, or macOS system using a system-wide
+hypervisor, such as VirtualBox, KVM/QEMU, VMware Fusion or Hyper-V. Using it is
+a popular way to test Kubernetes application locally prior to deploying it on a
+cloud.
+
+Following steps are needed to run PSMDB Operator on minikube:
+
+0. `Install minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_, using a way recommended for your system. This includes the installation of the following three components:
+   #. kubectl tool,
+   #. a hypervisor, if it is not already installed,
+   #. actual minikube package
+
+   After the installation running ``minikube start`` should download needed
+   virtualized images, initialize and run the cluster. After minikube is
```

Comment:
Minikube needs consistent capitalization in this document.  I believe it should be capitalized as it's a proper name, which means it should always be capitalized unless preformat quoting a code block containing a command which is lowercase.